api/src/main/java/org/pmiops/workbench/elasticsearch/ElasticFilters.java

```diff
@@ -241,7 +322,9 @@ private static boolean isStandardConcept(SearchParameter param) {
                 .map(id -> Long.toString(id))
                 .collect(Collectors.toSet()));
       } else {
-        out.add(Long.toString(param.getConceptId()));
+        if (param.getConceptId() != null) {
+          out.add(Long.toString(param.getConceptId()));
```

Comment:
Same as above - is it legal for a search parameter to correspond to 0 matching concept IDs?